test/corpus/functions.txt

```diff
@@ -1082,5 +1136,5 @@ io.println("// hello world!\n")
     arguments: (arguments
       (argument
         value: (string
-	  (quoted_content)
-	  (escape_sequence))))))
+    (quoted_content)
+    (escape_sequence))))))
```

Comment:
While we're talking about indentation, these two lines should not have changed :grin:

Reply:
these 2 lines had tabs and do not have tabs anymore.